torch/CMakeLists.txt

```diff
@@ -210,6 +210,18 @@ add_custom_command(
     WORKING_DIRECTORY
     "${TORCH_ROOT}"
 )
+file(GLOB_RECURSE datapipe_files "${TORCH_SRC_DIR}/utils/data/datapipes/*.py")
+add_custom_command(
```

Comment:
Hi @malfet, I added this new custom command here but it doesn't seem to be running when I run `python setup.py develop`. Any idea why that might be?

I tried a few things: should I wrap double quote around `${datapipe_files}`? Or maybe I'm using `GLOB_RECURSE` or its output improperly?

cc: @albanD

Reply:
I went through this again. Adding the output file as a dependency on L193 will make it run as needed.

This is now working as intended.